tempesta_fw/http_parser.c

```diff
@@ -644,7 +641,7 @@ good_looking_eol:							\
 	 * The next chunk of chunked-body payload (RFC 7320, 4.1.2)	\
 	 * needs to be handled.						\
 	 */								\
-	__FSM_JMP(RGen_Body);						\
+	__FSM_JMP(RGen_BodyChunk);					\
```

Comment:
I didn't get the logic. It seems at this point you don't know transfer encoding, we only do the check at `RGen_Body` state. At the state we check TFW_HTTP_CHUNKED and jump to `RGen_BodyChunk`... Why do you directly jump to `RGen_BodyChunk`?